### ISSUE TITLE
Add keyboard shortcuts

### ISSUE THREAD

**Issue by ComFreek** [yishn/tikzcd-editor#39]
When within one node, I'd suggest that pressing one of the arrow keys navigates the user to the desired node.

Maybe Ctrl-arrow key also adds an edge.

**Comment by yishn** [comment]
I'm not sure how you envision the keyboard shortcuts. When you're within one node, you're inside a text box, editing the LaTeX code for that node. Pressing arrow keys should move the caret of that text box.

**Comment by ComFreek** [comment]
You're right, arrow keys on their own would break normal editing. How about `Ctrl-arrow key` then for moving around from node to node and possibly `Ctrl-Alt-arrow-key` for moving while creating arrows?

I would have first suggested using `Tab`, but then this would only support going to the right.
